src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -7,14 +7,1284 @@
+     * `0` - reactive energy accumulation by quadrants `Q1, Q2, Q3, Q4`;
+     * `1` - reactive energy accumulation by `R+, R-`.
+     *
+     * Since build 302.19.XXX (XXX - decimal number) for G-type meters, the energies `R+` and `R-` are accumulated and displayed similarly to R-type meters.
+     * Reactive energy by quadrants is not accumulated.
+     */
+    ACCUMULATE_BY_R_PLUS_MINUS: boolean;
+}
+
+export interface IOperatorParameters {
+    /**
+     * Maximum voltage threshold, mV.
+     */
+    vpThreshold: types.TUint32,
+
+    /**
+     * Minimum voltage threshold, mV.
+     */
+    vThreshold: types.TUint32,
+
+    /**
+     * Maximum current threshold, mA.
+     */
+    ipThreshold: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T1`, Watts.
+     */
+    pmaxThreshold0: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T2`, Watts.
+     */
+    pmaxThreshold1: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T3`, Watts.
+     */
+    pmaxThreshold2: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T4`, Watts.
+     */
+    pmaxThreshold3: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T1`, VAR.
+     */
+    rmaxThreshold0: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T2`, VAR.
+     */
+    rmaxThreshold1: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T3`, VAR.
+     */
+    rmaxThreshold2: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T4`, VAR.
+     */
+    rmaxThreshold3: types.TUint32,
+
+    /**
+     * Power averaging interval, in minutes (1-60).
+     */
+    tint: types.TUint8,
+
+    /**
+     * Start date of the monthly billing period.
+     */
+    calcPeriodDate: types.TUint8,
+
+    /**
+     * Display active time.
+     */
+    timeoutDisplay: types.TUint8,
+
+    /**
+     * Display active time for each screen.
+     */
+    timeoutScreen: types.TUint8,
+
+    /**
+     * Display settings for meter readings (first 32 screens).
+     */
+    displaySet1: IDisplaySet1OperatorParameter,
+
+    /**
+     * Display settings for meter readings.
+     */
+    displaySet2: IDisplaySet2OperatorParameter,
+
+    /**
+     * Display settings for meter readings.
+     */
+    displaySet3: IDisplaySet3OperatorParameter,
+
+    /**
+     * Relay settings.
+     */
+    relaySet: IRelaySetOperatorParameter,
+
+    /**
+     * Controls the data transmission speed for UART interfaces.
+     */
+    speedOptoPort: ISpeedOptoPortOperatorParameter,
+
+    /**
+     * Integration period for energy profiles.
+     * `15` - `15` minutes; `0`, `30` - `30` minutes; `60` - `60` minutes.
+     * since build 0.0.17
+     */
+    ten: types.TUint8,
+
+    /**
+     * Voltage averaging interval.
+     * `0`, `1`, `3`, `5`, `10`, `15`, `30` minutes.
+     * since build 0.0.17
```

Comment:
```suggestion
     *
     * since build `0.0.17`
```

it should be distinctly visible
maybe even make "since" bold

Reply:
applied (without bold)